core/src/subgraph/instance_manager.rs

```diff
@@ -179,10 +162,8 @@ impl SubgraphInstanceMetrics {
         }
     }
 
-    pub fn observe_trigger_processing_duration(&self, duration: f64, trigger: TriggerType) {
-        self.trigger_processing_duration
-            .with_label_values(vec![trigger.label_value()].as_slice())
-            .observe(duration);
```

Comment:
This was a functional change, I don't think this label is very useful so it wasn't worth abstracting it, instead it was removed.

Reply:
I wonder if we could just get rid of the entire metric - I think the data we get out of the various stopwatch sections is much more useful.

Reply:
Could be, but for the purpose of this PR removing the label is enough.